tests/libsecret/secret.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package libsecret
+
+import (
+	kubev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// secretLabel set this label to make the test suite namespace clean-up delete the secret on teardown
```

Comment:
why cannot the cleanup delete all secrets? would there be any non-test secret that we want to keep in the test namepsace?

(exposing a public constant is a reason to worry and to ask questions)

Reply:
> why cannot the cleanup delete all secrets? would there be any non-test secret that we want to keep in the test namepsace?

I am not sure, this is why I didn't remove the filtering of the test suite namespace cleanup, there might be tests that relay on Secrets that remain between tests.
To answer that I need to go over all tests that involve secret and realize if they do depend on a Secret that created by other test.
I prefer to do that on a follow up PR, including verifying that no  secrets object is reused between tests.

Note that this PR doesn't introduce this global, it  moves it to related package.

Reply:
I mixed with the follow-up PR, sorry.

You right, for this PR there it make no sense to have `secretLabel` as package global, I moved it to New() scope.

In a the follow-up PR `secretLabel` will replace the one that in `tests/util/labels`.

Reply:
Let discuss removing SecretLabel entirely at the follow up PR #11547